front/pages/w/[wId]/assistant/[cId]/index.tsx

```diff
@@ -31,15 +31,16 @@ export const getServerSideProps: GetServerSideProps<{
   conversationId: string;
 }> = async (context) => {
   const session = await getSession(context.req, context.res);
-  const user = await getUserFromSession(session);
   const auth = await Authenticator.fromSession(
     session,
     context.params?.wId as string
   );
 
   const owner = auth.workspace();
+  const user = auth.user();
   const subscription = auth.subscription();
-  if (!owner || !auth.isUser() || !user || !subscription) {
+
+  if (!owner || !user || !auth.isUser() || !subscription) {
     return {
       redirect: {
         destination: `/w/${context.query.wId}/join?cId=${context.query.cId}`,
```

Comment:
@flvndvd is this flow compatible with your recent work on auto-join?

I presume yes but figured it was worth a ping since this is where we redirect all users clicking on a Dust conversation without being signed-in?

So it's a pretty important flow for team penetration.

Reply:
Thanks thanks!